lib/src/diff.rs

```diff
@@ -42,21 +46,29 @@ pub fn find_line_ranges(text: &[u8]) -> Vec<Range<usize>> {
     ranges
 }
 
-fn is_word_byte(b: u8) -> bool {
+fn is_word_grapheme(g: &str) -> bool {
     // TODO: Make this configurable (probably higher up in the call stack)
-    matches!(b, b'A'..=b'Z' | b'a'..=b'z' | b'0'..=b'9' | b'_')
+    let word_graphemes: HashSet<&str> =
+        vec!["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M",
+             "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z",
+             "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m",
+             "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", 
+             "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "_"]
+        .into_iter().collect();
+
+    word_graphemes.contains(g)
```

Comment:
I think you can leave this function mostly unchanged by doing `matches!(g.as_bytes()[0], ...)`

Reply:
Done!